packages/sync/src/Replicastore.php

```diff
@@ -1202,6 +1202,38 @@ public function get_checksum_columns_for_object_type( $object_type ) {
 		}
 	}
 
+	/**
+	 * Grabs the minimum and maximum object ids for the given parameters.
+	 *
+	 * @access public
+	 *
+	 * @param int    $id_field     The id column in the table to query.
+	 * @param bool   $distinct     If true, will query for a distinct object id.
+	 * @param string $object_table The table to query.
+	 * @param string $where        A sql where clause without 'WHERE'.
+	 * @param int    $bucket_size  The maximum amount of objects to include in the query.
+	 *
+	 * @return object An object with min_id and max_id properties.
+	 */
+	public function get_min_max_object_id( $id_field, $distinct, $object_table, $where, $bucket_size ) {
+		global $wpdb;
+
+		$distinct_sql = $distinct ? 'DISTINCT' : '';
```

Comment:
Can we infer the `$distinct` automatically from the `$object_table`? We have a limited amount of tables we expect here anyway.

Reply:
Yes!